docs/website/docs/dlt-ecosystem/verified-sources/index.md

```diff
@@ -6,14 +6,18 @@ keywords: ['verified source']
 import DocCardList from '@theme/DocCardList';
 import Link from '../../_book-onboarding-call.md';
 
-Pick one of our verified sources that we wrote or maintain ourselves. All of them are constantly tested on real data and distributed as simple Python code so they can be easily customized or hacked.
+Choose from our collection of verified sources, which dlt team developed and maintains. Each source is rigorously tested with real data and provided as Python code for easy customization.
```

Comment:
>dlt team

and dlt community :D

Reply:
> real

should we get rid of it? or our tests data is a real data?

Reply:
> real

I think what was initially intended is that it's real APIs (not real data) vs mock tests. I agree we need to rephrase that.